DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py

```diff
@@ -294,55 +294,55 @@
 if (process.runType.getRunType() == process.runType.pp_run or process.runType.getRunType() == process.runType.pp_run_stage1):
     #event selection for pp collisions
     if ((process.runType.getRunType() == process.runType.pp_run) and live):
-        process.source.SelectEvents = cms.untracked.vstring(
+        process.source.SelectEvents = [
             'HLT_L1*',
             'HLT_Jet*',
             'HLT_Physics*',
             'HLT_ZeroBias*',
             'HLT_PAL1*',
             'HLT_PAZeroBias*',
             'HLT_PAAK*'
-            )
+            ]
 
     # Source and Client config for pp collisions
 
-    process.SiStripMonitorDigi.UseDCSFiltering = cms.bool(False)
-    process.SiStripMonitorClusterReal.UseDCSFiltering = cms.bool(False)
+    process.SiStripMonitorDigi.UseDCSFiltering = False
+    process.SiStripMonitorClusterReal.UseDCSFiltering = False
 
     process.MonitorTrackResiduals_gentk.Tracks                 = 'initialStepTracksPreSplitting'
     process.MonitorTrackResiduals_gentk.trajectoryInput        = 'initialStepTracksPreSplitting'
-    process.MonitorTrackResiduals_gentk.TrackProducer          = cms.string('initialStepTracksPreSplitting')
-    process.TrackMon_gentk.TrackProducer    = cms.InputTag('initialStepTracksPreSplitting')
-    process.TrackMon_gentk.allTrackProducer = cms.InputTag('initialStepTracksPreSplitting')
+#    process.MonitorTrackResiduals_gentk.TrackProducer          = 'initialStepTracksPreSplitting' ##DOES NOT EXIST ON DQM/TrackerMonitorTrack/src/MonitorTrackResiduals.cc
+    process.TrackMon_gentk.TrackProducer    = 'initialStepTracksPreSplitting'
+    process.TrackMon_gentk.allTrackProducer = 'initialStepTracksPreSplitting'
     process.SiStripMonitorTrack_gentk.TrackProducer = 'initialStepTracksPreSplitting'
 
     process.SiStripSources_TrkReco   = cms.Sequence(process.SiStripMonitorTrack_gentk*process.MonitorTrackResiduals_gentk*process.TrackMon_gentk)
 
     ### STRIP
     process.load("DQM.SiStripMonitorClient.SiStripClientConfigP5_cff")
-    process.SiStripAnalyser.UseGoodTracks  = cms.untracked.bool(True)
+#    process.SiStripAnalyser.UseGoodTracks  = True   #DOES NOT EXIST ON DQM/SiStripMonitorClient/plugins/SiStripAnalyser.cc
```

Comment:
@mtosi @mmusich if you could please confirm? Thanks!

Reply:
the change *per se* looks OK (please remove the commented code as well), though the module is not cloned here while in the PR description there's written:
> ... "hidden" non-existing/wrong parameters or not actually declared in the original cloned module, have been identified in the process

by the way it seems there is a parameter with the same name configured here:

https://github.com/cms-sw/cmssw/blob/ac5ca816d5982e089eca9e7d0aec24c517922ced/DQM/SiStripMonitorClient/python/SiStripClientConfig_Tier0_cff.py#L11

and it doesn't seem to be registered in any source code:

https://cmssdt.cern.ch/dxr/CMSSW/search?q=UseGoodTracks&case=true&redirect=true

Reply:
Thanks!